sys/include/event/periodic.h

```diff
@@ -76,13 +82,27 @@ void event_periodic_init(event_periodic_t *event_periodic, ztimer_clock_t *clock
  * @param[in]   event_periodic   event_timout context object to use
  * @param[in]   interval         period length for the event
  */
-static inline void event_periodic_start(event_periodic_t *event_periodic,
-                                        uint32_t interval)
+static inline void event_periodic_start(event_periodic_t *event_periodic, uint32_t interval)
```

Comment:
Is `event_periodic->count` uninitialized here?

Reply:
No, since `event_periodic_init` needs to be called before on any `event_periodic`.